### PR TITLE
Schema browser refresh on connection

### PR DESCRIPTION
**Pull request by shamby87**

### CODE REVIEW

File: src/views/schemaBrowser.ts [hunk 1]
```diff
@@ -123,18 +123,25 @@ export default class schemaBrowser {
...
+            await getInstance().setConfig(config);
           }
 
           vscode.window.showInformationMessage(`Current schema set to ${schema}.`);
         }
       })
     )
+
+    getInstance().onEvent(`connected`, () => {
+      fetchSystemInfo().then(() => {
```
Comment (worksofliam) [inline src/views/schemaBrowser.ts:140]: The call to `fetchSystemInfo` is not required here.

**Review by worksofliam — CHANGES_REQUESTED** [review]
Check out my comment. Should be an easy change, as it is not required here.

File: src/views/schemaBrowser.ts (hunk 1 above)
Comment (shamby87) [reply src/views/schemaBrowser.ts]: Fixed

**Comment by shamby87** [comment]
Yep, I was going to try that and forgot. Should be good now

**Pull request merged**